presale/models/presale_order.py

```diff
@@ -0,0 +1,89 @@
+from odoo import api, fields, models
+
+
+class PresaleOrder(models.Model):
+    _name = 'presale.order'
+    _description = 'presales orders'
+
+    name = fields.Char(string='Preorder Reference',
+                       required=True, copy=False, readonly=True,
+                       default=lambda self: 'New')
```

Comment:
No need for a lambda function if there is no calculation made
```suggestion
                       default='New')
```